src/zabapgit_unit_test.prog.abap

```diff
@@ -1793,6 +1811,90 @@ CLASS ltcl_html_action_utils IMPLEMENTATION.
 
   ENDMETHOD.  "get_field
 
+  METHOD parse_fields_simple_case.
+
+    _given_string_is( `committer_name=Gustav Gans` ).
+
+    _when_fields_are_parsed( ).
+
+    _then_fields_should_be( index = 1 name = `COMMITTER_NAME` value = `Gustav Gans` ).
+
+  ENDMETHOD.
+
+  METHOD parse_fields_advanced_case.
+
+    _given_string_is( `committer_name=Albert Schweitzer&`
+                   && `committer_email=albert.schweitzer@googlemail.com&`
+                   && `comment=dummy comment&`
+                   && `body=Message body<<new>><<new>>with line break<<new>>&`
+                   && `author_name=Karl Klammer&`
+                   && `author_email=karl@klammer.com` ).
+
+    _when_fields_are_parsed( ).
+
+    _then_fields_should_be(:
+      index = 1 name = `COMMITTER_NAME`   value = `Albert Schweitzer` 															 ),
+      index = 2 name = `COMMITTER_EMAIL`  value = `albert.schweitzer@googlemail.com`								 ),
+      index = 3 name = `COMMENT`          value = `dummy comment` 																	 ),
+      index = 4 name = `BODY`             value = `Message body<<new>><<new>>with line break<<new>>` ),
+      index = 5 name = `AUTHOR_NAME`      value = `Karl Klammer`																		 ),
+      index = 6 name = `AUTHOR_EMAIL`     value = `karl@klammer.com`																 ).
```

Comment:
yep, it does compile. In my opinion in this case the colon operator leads to cleaner code. Anyway I unchain the method calls. What worries me more is that I can't use non 7 bit ASCII characters in my unit tests to test the german umlauts. Any suggestions about this?